src/pages/Home.vue

```diff
@@ -1,6 +1,122 @@
 <template>
   <main>
-    <h1>LevelDB Inspector</h1>
-    <img src="@images/logo.png" alt="App Logo" />
+    <header>
+      <div class="logo"></div>
+    </header>
+    <form>
+      <p>Select path to LevelDB</p>
+      <label class="file db-path">
+        <input type="file" id="file" aria-label="File browser example" />
```

Comment:
@jjlumagbas I think we need to choose a LevelDB directory, instead of accepting a file (can you confirm @emanlodovice ?) We might have to use Electron's [`dialog` module](https://www.electronjs.org/docs/api/dialog) to do this, since it allows selecting a directory.

It needs to be called from the main process though, but since the directory path is used in the main process anyway then it's better that way. So the purpose of the UI would then be just to call a function in the main process (via IPC), then the main process can take care of choosing the directory and sending that directory back to the renderer process (via IPC again), then the UI can just display the chosen path in the input field. What do you think? 😅

Reply:
@arnellebalane perhaps more straightforward is:

https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitdirectory

and

https://stackoverflow.com/questions/38530293/electron-get-full-path-of-uploaded-file

?

Reply:
@jjlumagbas Ohhh that is better indeed! Didn't know Electron modifies that standard behavior 😓